ironfish-cli/src/commands/accounts/notes.ts

```diff
@@ -37,9 +37,12 @@ export class NotesCommand extends IronfishCommand {
         [note],
         {
           amount: {
-            header: 'Amount ($IRON)',
+            header: 'Amount',
             get: (row) => CurrencyUtils.renderIron(row.amount),
           },
+          assetIdentifier: {
```

Comment:
This isn't that valuable we should show the name here and not the identifier. It's fine to keep this but keep it hidden by default like we do in Peers list.

Reply:
The name isn't unique, therefore should not be used as a source of truth in situations like this. The identifier is the only way to truthfully show which asset is which - much like smart contracts have to rely on their addresses

Reply:
Agreed with @mat-if. I'd probably show both

Reply:
My comment was to show both. The CLI here should show the name, but nothing accepts the name so they'll have to get the identifier and pass it in.

Reply:
I will show both.